src/main/java/edu/unc/lib/boxc/migration/cdm/status/SourceFilesSummaryService.java

```diff
@@ -41,14 +41,24 @@ public void summary(Verbosity verbosity) {
         int totalFilesMapped = totalFilesMapped();
         int newFilesMapped = totalFilesMapped - previousStateFilesMapped;
         int totalObjects = totalFilesInProject();
-        List<CSVRecord> listFiles = listNewFiles();
+        List<CSVRecord> listSampleFiles = listNewFiles();
+        List<CSVRecord> listAllFiles = listNewlyPopulatedFiles();
```

Comment:
Based off my reading of the ticket, i think we want to display the full contents of the new mapping rather than just the new populated fields, but you could check with morgan to be sure